Dockerfile

```diff
@@ -3,8 +3,8 @@
 #     CGO_ENABLED=0 go build -a -tags netgo
 #     docker build --rm=true -t plugins/drone-bintray .
 
-FROM gliderlabs/alpine:3.1
-RUN apk-install ca-certificates
-RUN apk-install curl
+FROM alpine:3.3
+RUN apk update && apk add ca-certificates
 ADD drone-bintray /bin/
+COPY ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
```

Comment:
No, this was my problem.  After making the previous comment, I remembered that I tested Alpine's certificate before adding the Go code to append it to the pool.  I retested, and this works with the Alpine cert.  I've removed the extra certificate from the merge.